docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -348,6 +348,310 @@ and a file system location of ``/var/my_tables/test_table``::
+
+To retrieve the information about the data files of the Iceberg table ``test_table`` use the query::
+
+    SELECT * FROM "test_table$files"
+
+.. code-block:: text
+
+     content  | file_path                                                                                                                     | record_count    | file_format   | file_size_in_bytes   |  column_sizes        |  value_counts     |  null_value_counts | nan_value_counts  | lower_bounds                |  upper_bounds               |  key_metadata  | split_offsets  |  equality_ids
+    ----------+-------------------------------------------------------------------------------------------------------------------------------+-----------------+---------------+----------------------+----------------------+-------------------+--------------------+-------------------+-----------------------------+-----------------------------+----------------+----------------+---------------
+     0        | hdfs://hadoop-master:9000/user/hive/warehouse/test_table/data/c1=3/c2=2021-01-14/af9872b2-40f3-428f-9c87-186d2750d84e.parquet |  1              |  PARQUET      |  442                 | {1=40, 2=40, 3=44}   |  {1=1, 2=1, 3=1}  |  {1=0, 2=0, 3=0}   | <null>            |  {1=3, 2=2021-01-14, 3=1.3} |  {1=3, 2=2021-01-14, 3=1.3} |  <null>        | <null>         |   <null>
+
+
+
+The output of the query has the following columns:
+
+.. list-table:: Files columns
+  :widths: 20, 30, 50
+  :header-rows: 1
+
+  * - Name
+    - Type
+    - Description
+  * - ``content``
+    - ``integer``
+    - Type of content stored in the file.
+      The supported types of content supported in Iceberg are
+
+      * ``DATA(0)``
+      * ``POSITION_DELETES(1)``
+      * ``EQUALITY_DELETES(2)``
+  * - ``file_path``
+    - ``varchar``
+    - The data file location
+  * - ``file_format``
+    - ``varchar``
+    - The format of the data file
+  * - ``record_count``
+    - ``bigint``
+    - The number of entries contained in the data file
+  * - ``file_size_in_bytes``
+    - ``bigint``
+    - The data file size
+  * - ``column_sizes``
+    - ``map(integer, bigint)``
+    - The size for each column in the columnar file
+  * - ``value_counts``
+    - ``map(integer, bigint)``
+    - The count of entries per column in columnar file
+  * - ``null_value_counts``
+    - ``map(integer, bigint)``
+    - The number of ``NULL`` values per column in the columnar file
+  * - ``nan_value_counts``
+    - ``map(integer, bigint)``
+    - The number of non numerical values per column in the columnar file
+  * - ``lower_bounds``
+    - ``map(integer, bigint)``
+    - The lower bounds per column in the columnar file
+  * - ``upper_bounds``
+    - ``map(integer, bigint)``
+    - The upper bounds per column in the columnar file
+  * - ``key_metadata``
+    - ``varbinary``
+    - Metadata about how this file is encrypted, if applicable
```

Comment:
So this is an odd one, as it's not currently used in open source (though it's coming and a high priority on the roadmap).

The javadoc for this interface (which is a light wrapper around a ByteBuffer) refer to this as `metadata about an encrypted data file's encryption key.` More specifically, it likely refers to the location of the encryption key but that's probably being iterated on to be more efficient so I wouldn't state that for now.

Maybe `Metadata about the encryption key used to encrypt this file, if applicable`?

I should mention that the [avro doc comment for this just simply says `Encryption key metadata blob`](**https://github.com/apache/iceberg/blob/461cd7b1e90c589f411d739b24b5c34848e3cdd5/api/src/main/java/org/apache/iceberg/DataFile.java#L59**).